gpib_server.py

```diff
@@ -194,6 +195,20 @@ def query(self, c, data):
         ans = instr.read_raw()
         return str(ans).strip()
 
+    @setting(7, n_bytes='w', returns='s')
```

Comment:
Should use the new bytes type "y" for the return type here, since this can be binary data. We'll have to think about modifying other settings in this server, but would be good to get this one right.